docset/winserver2022-ps/addsdeployment/Install-ADDSDomainController.md

```diff
@@ -627,8 +722,12 @@ This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVar
 ### Microsoft.DirectoryServices.Deployment.Types.Result
 
 ## NOTES
-* By default, this cmdlet always prompts for confirmation. To bypass confirmation, you need to include the **Confirm** parameter and specify a value of `$false`. For example, `-Confirm:$false`.
-* By default, this cmdlet is always run when executed. To see what will happen if the cmdlet runs without executing or committing installation changes, first run the cmdlet using the ***WhatIf** parameter to show what would happen.
+
+- By default, this cmdlet always prompts for confirmation. To bypass confirmation, you need to
+  include the _Confirm_ parameter and specify a value of `$false`. For example, `-Confirm:$false`.
+- By default, this cmdlet is always run when executed. To see what will happen if the cmdlet runs
+  without executing or committing installation changes, first run the cmdlet using the _WhatIf_
+  parameter to show what would happen.
```

Comment:
```suggestion
- By default, this cmdlet always prompts for confirmation. To bypass confirmation, you need to
  include the **Confirm** parameter and specify a value of `$false`. For example,
  `-Confirm:$false`.
- By default, this cmdlet is always run when executed. To see what will happen if the cmdlet runs
  without executing or committing installation changes, first run the cmdlet using the **WhatIf**
  parameter to show what would happen.
```

Reply:
@michaeltlombardi sorry about that, the Markdown linter keeps triggering this as a problem and suggesting the emphasis change: https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md049.md

Since that doesn't line up with the PowerShell style guide, do you know if that is something I can change in VS Code so it matches what we're wanting?